docs/source/export.rst

```diff
@@ -1,11 +1,293 @@
+  appearing in the graph. In this case, we see that ``s0`` and ``s1`` have
+  the range [2, inf]. For technical reasons that are difficult to explain here,
+  they are assumed to be not 0 or 1. This is not a bug, and does not necessarily
+  mean that the exported program will not work for dimensions 0 or 1. See
+  `The 0/1 Specialization Problem <https://docs.google.com/document/d/16VPOa3d-Liikf48teAOmxLc92rgvJdfosIy-yoT38Io/edit?fbclid=IwAR3HNwmmexcitV0pbZm_x1a4ykdXZ9th_eJWK-3hBtVgKnrkmemz6Pm5jRQ#heading=h.ez923tomjvyk>`_
+  for an in-depth discussion of this topic.
+
+* ``exported_program.equality_constraints`` describes which dimensions are
+  required to be equal. Since we specified in the constraints that the first
+  dimension of each argument is equivalent,
+  (``dynamic_dim(example_args[0], 0) == dynamic_dim(example_args[1], 0)``),
+  we see in the equality constraints the tuple specifying that ``arg4_1``
+  dimension 0 and ``arg5_1`` dimension 0 are equal.
+
+
+Limitations of torch.export
+---------------------------
+
+Graph Breaks
+^^^^^^^^^^^^
+
+As ``torch.export`` is a one-shot process for capturing a computation graph from
+a PyTorch program, it might ultimately run into untraceable parts of programs as
+it is nearly impossible to support tracing all PyTorch and Python features. In
+the case of ``torch.compile``, an unsupported operation will cause a "graph
+break" and the unsupported operation will be run with default Python evaluation.
+In contrast, ``torch.export`` will require users to provide additional
+information or rewrite parts of their code to make it traceable. As the
+tracing is based on TorchDynamo, which evaluates at the Python
+bytecode level, there will be significantly fewer rewrites required compared to
+previous tracing frameworks.
+
+When a graph break is encountered, :ref:`ExportDB <torch.export_db>` is a great
+resource for learning about the kinds of programs that are supported and
+unsupported, along with ways to rewrite programs to make them traceable.
+
+Control Flow
+^^^^^^^^^^^^
+
+Graph breaks can also be encountered on general control flow
+(ex. ``if x[0] > 4``), which a tracing compiler cannot possibly deal with
+without generating code for a combinatorially exploding number of paths. In such
+cases, users will need to rewrite their code using special
+**Control Flow Operators**.
+
+Data-Dependent Accesses
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Data dependent behavior such as using the value inside of a tensor to construct
+another tensor, or using the value of a tensor to slice into another tensor, is
+also something the tracer cannot fully determine. Users will need to rewrite
+their code using the inline constraint APIs
+:func:`torch.export.constrain_as_size` and
+:func:`torch.export.constrain_as_value`.
+
+Missing Meta Kernels for Operators
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When tracing, a META implementation (or "meta kernel") is required for all
```

Comment:
depending on who the audience is for this, it's helpful to separate out content for export devs vs end users